### PR TITLE
use preferred skin tone emoji when add using enter

### PR DESCRIPTION
**Pull request by thesahindia**
### Details

Fixed the default emoji issue when added using enter

### Fixed Issues

$ https://github.com/Expensify/App/issues/7264

### Tests

- [x] Verify that no errors appear in the JS console

### QA Steps

1. Open a chat
2. Open the emoji picker
3. Change the default skin tone to something other than yellow
4. Select an emoji and hit enter/return on the keyboard
5. Verify the added emoji's skin tone is the preferred skin tone
- [x] Verify that no errors appear in the JS console

### Tested On

- [x] Web
- [ ] Mobile Web
- [x] Desktop
- [ ] iOS
- [ ] Android

#### Web

https://user-images.githubusercontent.com/83179295/150001950-4eb24f4d-1e45-48ba-8415-2275e25d2885.mov

#### Desktop

https://user-images.githubusercontent.com/83179295/150001966-0bfbc41a-56c0-430b-8f89-dd098fb1ec05.mov

### CODE REVIEW

File: src/pages/home/report/EmojiPickerMenu/index.js [hunk 1]
```diff
@@ -142,7 +142,11 @@ class EmojiPickerMenu extends Component {
 
             // Select the currently highlighted emoji if enter is pressed
             if (keyBoardEvent.key === 'Enter' && this.state.highlightedIndex !== -1) {
-                this.props.onEmojiSelected(this.state.filteredEmojis[this.state.highlightedIndex].code, this.state.filteredEmojis[this.state.highlightedIndex]);
+                const item = this.state.filteredEmojis[this.state.highlightedIndex];
+                const emoji = item.types && item.types[this.props.preferredSkinTone]
```
Comment (parasharrajat) [inline src/pages/home/report/EmojiPickerMenu/index.js:146]: use lodashGet instead.

**Review by parasharrajat — CHANGES_REQUESTED** [review]

**Review by luacmartins — CHANGES_REQUESTED** [review]
What @parasharrajat said

File: src/pages/home/report/EmojiPickerMenu/index.js (hunk 1 above)
Comment (thesahindia) [reply src/pages/home/report/EmojiPickerMenu/index.js]: Done.

**Review by luacmartins — APPROVED** [review]
LGTM and tests well. Thanks @thesahindia!

**Review by parasharrajat — COMMENTED** [review]
LGTM.  a little late but I reviewed it already.

:ribbon: :eyes: :ribbon:  C+ reviewed

**Comment by hayata-suenaga** [comment]
I'll link a related [regression](https://github.com/Expensify/App/issues/15232) reported here, although this PR might have not directly caused the issue.

There is a possibility that emoji is not highlighted when there is no emoji candidates. When that happens `emoji` is `undefined`, and accessing `code` property of `undefined` causes an error.

**Pull request merged**